<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonGo">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>328</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Go!</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>600</width>
      <height>180</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableWidget" name="tableWidgetA">
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <row/>
      <column/>
     </widget>
     <widget class="QSpinBox" name="spinBoxA">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableWidget" name="tableWidgetB">
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <row/>
      <column/>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="spinBoxB">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>195</y>
      <width>86</width>
      <height>26</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QSpinBox" name="spinBoxElement"/>
    <widget class="QLabel" name="labelElement">
     <property name="text">
      <string>HERE</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_5">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>227</y>
      <width>145</width>
      <height>17</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Power  Plural A:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPowerA">
     <property name="text">
      <string>HERE</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_6">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>250</y>
      <width>192</width>
      <height>17</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>MAX Element Plural A:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMaxElementA">
     <property name="text">
      <string>HERE</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_7">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>273</y>
      <width>187</width>
      <height>17</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>MIN Element Plural A:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMinElementA">
     <property name="text">
      <string>HERE</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_8">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>296</y>
      <width>215</width>
      <height>26</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>DeleteElement  Plural A:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxDeleteElment"/>
   </widget>
   <widget class="QSplitter" name="splitter_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>601</width>
      <height>401</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter_9">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Plura (A + B)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="tableWidgetAdding">
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <row/>
      <column/>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_10">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Plura (A - B)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="tableWidgetMinus">
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <row/>
      <column/>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_11">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget_3">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Plura (A / B)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="tableWidgetDevide">
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <row/>
      <column/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxA</sender>
   <signal>valueChanged(int)</signal>
   <receiver>tableWidgetA</receiver>
   <slot>resizeColumnToContents(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
